core/dbt/logger.py

```diff
@@ -323,8 +323,6 @@ def process(self, record):
 
 warnings.filterwarnings("ignore", category=ResourceWarning, message="unclosed.*<socket.socket.*>")
 
-initialized = False
```

Comment:
What was this? Are we sure it is unused?

Reply:
Yep.  It's for RPC.. putting it back.

Reply:
Yep.  It's needed by dbt-rpc.  Putting it back.
:face_palm: